site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-list.vm

```diff
@@ -1,5 +1,93 @@
 ##<!-- $Header: /cvs/sakai2/legacy/tools/src/webapp/vm/sitesetup/chef_site-siteInfo-list.vm,v 1.6 2005/05/28 03:04:36 ggolden.umich.edu Exp $ -->
 
+## This macro has been defined in order to reuse the paging panel once at the top of the participant group table and once at the bottom of the participant group table. JIRA Ref -WL3736
```

Comment:
Could you not include WL-ref comments in your code as this code should go back to Sakai and local references won't mean anything there.